src/index.js

```diff
@@ -1,6 +1,30 @@
 import './styles.css';
-import { scoreList, updateList } from './modules/scorelist';
+import updateList from './modules/scorelist';
+import { getScore, postScore } from './modules/generate_api';
 
 const scoreBoard = document.getElementById('scoreboard');
+const refreshBtn = document.getElementById('refresh-btn');
+const submitForm = document.querySelector('.input-score');
+const player = document.getElementById('name');
+const score = document.getElementById('score');
 
-updateList(scoreList, scoreBoard);
\ No newline at end of file
+async function updateScore() {
+  const fetchAPI = await getScore();
+  const updatedList = fetchAPI.result;
+  updatedList.sort((a, b) => a.score - b.score);
+  updateList(updatedList);
+}
```

Comment:
- [ ] Kindly refactor this method to an arrow function.

Reply:
Thank you for the feedback